nix/hosts/webforge/configuration.nix

```diff
@@ -8,5 +8,32 @@
   networking.hostName = "webforge";
   networking.domain = "tahoe-lafs.org";
 
+  # Enable firewall with the required ports
+  networking.firewall = {
+    enable = true;
+    allowPing = false;
```

Comment:
My personal taste is to allow ping and also [rejectPackets](https://mynixos.com/nixpkgs/option/networking.firewall.rejectPackets) (to be a good internet citizen), like the OGs, in [this ticket](https://github.com/NixOS/nixpkgs/issues/12927) represented by fpletz and edolstra ;)

But I am OK with this choice as well.